pod.go

```diff
@@ -1220,3 +1229,63 @@ func (p *Pod) detachDevices() error {
 
 	return nil
 }
+
+func (p *Pod) updateResources() error {
```

Comment:
This should take a container as argument, so that it can be called from `createContainer()`.

Reply:
I was expecting this to be called every time we stop a container, am I missing something ?

Reply:
yep, you're right, adding it